src/AppHost.ts

```diff
@@ -1,73 +1,69 @@
 export interface IAppHost {
     /**
-     * Generates a profile of media the app can stream
+     * Queries the host application for the name of the app.
      *
-     * @param profileBuilder A helper that makes the profile creation easier
+     * @returns The name of the client.
      */
-    getDeviceProfile(profileBuilder: unknown): Promise<unknown>;
+    appName(): string;
 
     /**
-     * Generates a profile of media the app can sync
+     * Queries the host application for a the version of the app.
      *
-     * @param profileBuilder A helper that makes the profile creation easier
-     * @param appSettings TODO: Add description
-     */
-    getSyncProfile(
-        profileBuilder: unknown,
-        appSettings: unknown
-    ): Promise<unknown>;
```

Comment:
Why is this completly removed?

Reply:
Because it's completely unused in the web client, we have no sync.